TrustnetAuthenticWebQ.md

```diff
@@ -90,7 +98,7 @@ The chain of events for credibility assessment are:
 * Reputation. This can include provenance of the chain of providers, verified trust seals.
 * Transparency requires the information provider to disclose information about themselves and their content.
 
-Fundamentally, Trustnet provides a reputation signal for each facilitator and supports aggregation of assessments from
+Fundamentally, Trustnet provides a reputation signal for each piece of content based on an aggregation of assessments from

```

Comment:
I might just be misunderstanding this, but it seemed like the consumer rates the facilitators directly, so it's not Trustnet providing a reputation signal for them.